src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1612,6 +1613,16 @@ public SyntaxNode DottedName(string dottedName)
         public abstract SyntaxNode TypeExpression(ITypeSymbol typeSymbol);
 
         /// <summary>
+        /// Creates an expression that denotes a type.
+        /// Adds a <see cref="DoNotAddImportsAnnotation"/> which will prevent any
```

Comment:
ok.  so i still think i'd like it the reverse.  have the default state be to not add an import.  but allow people who do want imports to be addable to be able to provide that.  I do not like people having to go in and suppress things.